### PR TITLE
aws-c-cal: fix aws-c-common version on aws-c-cal/0.5.11

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **aws-c-cal/0.5.11**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
I can't find an explicit version dependency listed on https://github.com/awslabs/aws-c-cal/blob/c4c5b175e05f2affe5e3f0203ca6c8bc5cdd8f51/CMakeLists.txt#L125

Where did you find it? Is there a log showing the compatibility error?

**Comment by toge** [comment]
Sorry for the lack of information.
This pr is a part of https://github.com/conan-io/conan-center-index/issues/7763.

Compatibility error is shown in aws-c-io https://github.com/conan-io/conan-center-index/pull/7933.

```
WARN: aws-c-cal/0.5.11: requirement aws-c-common/0.6.15 overridden by aws-c-io/0.10.9 to aws-c-common/0.6.11 
ERROR: Missing binary: aws-c-cal/0.5.11:f44519c9c426b573d91175ee5df58f165e648480
aws-c-cal/0.5.11: WARN: Can't find a 'aws-c-cal/0.5.11' package for the specified settings, options and dependencies:
- Settings: arch=x86_64, build_type=Release, compiler=gcc, compiler.version=5, os=Linux
- Options: fPIC=True, shared=False, aws-c-common:cpu_extensions=True, aws-c-common:fPIC=True, aws-c-common:shared=False, openssl:386=False, openssl:enable_weak_ssl_ciphers=False, openssl:fPIC=True, openssl:no_aria=False, openssl:no_asm=False, openssl:no_async=False, openssl:no_bf=False, openssl:no_blake2=False, openssl:no_camellia=False, openssl:no_cast=False, openssl:no_chacha=False, openssl:no_cms=False, openssl:no_comp=False, openssl:no_ct=False, openssl:no_deprecated=False, openssl:no_des=False, openssl:no_dgram=False, openssl:no_dh=False, openssl:no_dsa=False, openssl:no_dso=False, openssl:no_ec=False, openssl:no_ecdh=False, openssl:no_ecdsa=False, openssl:no_engine=False, openssl:no_filenames=False, openssl:no_gost=False, openssl:no_hmac=False, openssl:no_idea=False, openssl:no_md4=False, openssl:no_md5=False, openssl:no_mdc2=False, openssl:no_ocsp=False, openssl:no_pinshared=False, openssl:no_rc2=False, openssl:no_rfc3779=False, openssl:no_rmd160=False, openssl:no_rsa=False, openssl:no_seed=False, openssl:no_sha=False, openssl:no_sm2=False, openssl:no_sm3=False, openssl:no_sm4=False, openssl:no_sock=False, openssl:no_srp=False, openssl:no_srtp=False, openssl:no_sse2=False, openssl:no_ssl=False, openssl:no_ssl3=False, openssl:no_stdio=False, openssl:no_tests=False, openssl:no_threads=False, openssl:no_tls1=False, openssl:no_ts=False, openssl:no_whirlpool=False, openssl:openssldir=None, openssl:shared=False
- Dependencies: aws-c-common/0.6.11, openssl/1.1.1l
- Requirements: aws-c-common/0.6.11, openssl/1.Y.Z
- Package ID: f44519c9c426b573d91175ee5df58f165e648480

ERROR: Missing prebuilt package for 'aws-c-cal/0.5.11'
Try to build from sources with '--build=aws-c-cal'
Use 'conan search <reference> --table table.html'
Or read 'http://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package'
```

This error seems to be caused by same reason in https://github.com/conan-io/conan-center-index/issues/7763.

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by Zvicii — APPROVED** [review]

**Review by Zvicii — APPROVED** [review]

File: recipes/aws-c-cal/all/conanfile.py [hunk 1]
```diff
@@ -35,6 +34,11 @@ def _source_subfolder(self):
     def _needs_openssl(self):
         return self.settings.os != "Windows" and not tools.is_apple_os(self.settings.os)
 
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
```
Comment (prince-chrismc) [inline recipes/aws-c-cal/all/conanfile.py:40]: Why was this changed? Does it save a few patches from being copied?

Comment (toge) [reply recipes/aws-c-cal/all/conanfile.py:40]: Yes, I have done this for other recipes as well.

All versions in this recipe require the same patch now, but this may change in the future, so the change was made for the purpose of omitting copying.

**Review by prince-chrismc — APPROVED** [review]

File: recipes/aws-c-cal/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/aws-c-cal/all/conanfile.py:40]: Thanks for confirming!
